examples/nlp_example.py

```diff
@@ -97,6 +98,8 @@ def collate_fn(examples):
 def training_function(config, args):
     # Initialize accelerator
     accelerator = Accelerator(cpu=args.cpu, mixed_precision=args.mixed_precision)
+    if accelerator.process_index == 1:
+        raise ValueError()
```

Comment:
I think this is a leftover of your tests?

Reply:
Thanks, fixed this now! 🤗